recipes/msys2/all/conanfile.py

```diff
@@ -142,7 +142,8 @@ def _do_build(self):
             for package in packages:
                 self.run(f'bash -l -c "pacman -S {package} --noconfirm"')
             for package in ['pkgconf']:
-                self.run(f'bash -l -c "pacman -Rs -d -d $(pacman -Qsq {package}) --noconfirm"')
+                if self.run(f'bash -l -c "pacman -Qs {package}"', ignore_errors=True) == 0:
+                    self.run(f'bash -l -c "pacman -Rs -d -d $(pacman -Qs {package}) --noconfirm"')
```

Comment:
```suggestion
                    self.run(f'bash -l -c "pacman -Rs -d -d {package} --noconfirm"')
```

I guess it should be updated to the package name as well.